build-tools/create-dotnet-pkg/create-dotnet-pkg.csproj

```diff
@@ -34,7 +34,7 @@
       <_FilesToCopy Include="$(DotNetPreviewPath)\sdk\$(DotNetPreviewVersionFull)\EnableWorkloadResolver.sentinel" />
       <_FilesToCopy Include="$(DotNetPreviewPath)\sdk-manifests\$(DotNetPreviewVersionBand)\Microsoft.NET.Workload.Android\**\*" />
       <_FilesToCopy Include="$(DotNetPreviewPath)\packs\Microsoft.Android.Ref\**\*" />
-      <_FilesToCopy Include="$(DotNetPreviewPath)\packs\Microsoft.Android.Sdk\**\*" />
+      <_FilesToCopy Include="$(DotNetPreviewPath)\packs\Microsoft.Android.Sdk.osx-x64\**\*" />
```

Comment:
This reminds me of a previous conversation around how "safe" it is to use `**\*` globs, as it could possibly pull in files that we don't expect ("build cruft", files unexpectedly copied from the system mono, etc.).

Should this instead be a more "constrained" glob expression and/or specific files?  See e.g. https://github.com/xamarin/xamarin-android/pull/2733#discussion_r258326881

Reply:
This wildcard is safe, as all the content in this path was extracted from `Microsoft.Android.Sdk.osx-x64.$(Version).nupkg`. All of the content being added to the SDK .nupkg file is explicitly defined through https://github.com/xamarin/xamarin-android/blob/master/build-tools/installers/create-installers.targets and https://github.com/xamarin/xamarin-android/blob/master/build-tools/create-packs/Microsoft.Android.Sdk.proj#L71.

This PR did fix one instance of us accidentally including extra build output in our packs by changing the output directory for Microsoft.Android.Sdk.ILLink.